proto/src/main/proto/Rss.proto

```diff
@@ -302,5 +304,4 @@ message GetShuffleDataStorageInfoResponse {
 message CheckServiceAvailableResponse {
   StatusCode status = 1;
   bool available = 2;
-}
-
+}
```

Comment:
need an empty line

Reply:
There is already a automatic waring here, you should fix it.

Reply:
I misunderstand the rule that you wanna express.